src/cli.ts

```diff
@@ -32,14 +41,19 @@ export function run(): Promise<void> {
     }
 
     if (cliOptions.help) {
-      var usage = cli.getUsage({
-        header: 'A development server for Polymer projects',
-        title: 'polyserve',
-      });
-      console.log(usage);
+      printUsage(cli);
+      reject();
```

Comment:
I think of it in the synchronous version:

```
/**
  * @options.help prints usage and throws
  * @returns a Server
  * @throws if option.help is true
 */
run(options) {}
```

We probably wouldn't this, and returning null might be ok. If you don't hate null too much.